login/logout.go

```diff
@@ -38,39 +40,54 @@ func (s *KeycloakLogoutService) Logout(ctx *app.LogoutLogoutContext, logoutEndpo
 	redirectURL, err := url.Parse(*redirect)
 	if err != nil {
 		log.Error(ctx, map[string]interface{}{
-			"redirect-url": redirectURL,
+			"redirect_url": redirectURL,
 			"err":          err,
 		}, "Failed to logout. Unable to parse redirect url.")
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrBadRequest(err.Error()))
 	}
 
-	redirectURLStr := redirectURL.String()
-	matched, err := regexp.MatchString(validRedirectURL, redirectURLStr)
+	osioRedirectURLStr := redirectURL.String()
+	matched, err := regexp.MatchString(validRedirectURL, osioRedirectURLStr)
 	if err != nil {
 		log.Error(ctx, map[string]interface{}{
-			"redirect-url":       redirectURLStr,
-			"valid-redirect-url": validRedirectURL,
+			"redirect_url":       osioRedirectURLStr,
+			"valid_redirect_url": validRedirectURL,
 			"err":                err,
 		}, "Can't match redirect URL and whitelist regex")
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal(err.Error()))
 	}
 	if !matched {
 		log.Error(ctx, map[string]interface{}{
-			"redirect-url":       redirectURLStr,
-			"valid-redirect-url": validRedirectURL,
+			"osio_redirect_url":  osioRedirectURLStr,
```

Comment:
What is osio_register_url?
I used osio_redirect_url just to avoid confusion since there are two redirect URLs now: 1. our original redirect to osio (which is logged here) and internal redirect to RHD.